haskell/test/Lorentz/Contracts/Test/Management.hs

```diff
@@ -301,7 +307,35 @@ managementSpec originate = do
 
         mgmContractPaused err
 
+  describe "Minter limit check" $ do
+    -- Currently the contract allows to add upto 12 minters.
```

Comment:
I don't like that 12 (or 13 in one place) is hardcoded in so many places (including comments). Can you define a constant in just one place?